wurst/network/Network.wurst

```diff
@@ -0,0 +1,506 @@
+package Network
+
+import SyncSimple
+import ErrorHandling
+import Execute
+import Buffer
+
+import initlater Metadata
+import initlater GamecacheKeys
+import initlater GamecacheBuffer
+import initlater StringEncoder
+
+/** 
```

Comment:
me personally, I prefer more compactz hotdoc with no empty lines. If u rpefer this you can leave it however.